CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and 
 [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+## [0.2.2] - 2022-12-16
++ Update - Requirements to accomodate DLC package requirements changing
++ Fix - Typos in docstrings
```

Comment:
Please follow the style in the rest of the doc. The [markdown lint](https://github.com/DavidAnson/markdownlint) extension can automate suggestions. 

```suggestion

## [0.2.2] - 2022-12-16

+ Update - Requirements to accomodate DLC package requirements changing
+ Fix - Typos in docstrings

```

Reply:
> Please follow the style in the rest of the doc. The [markdown lint](https://github.com/DavidAnson/markdownlint) extension can automate suggestions.

Updated it after installing the markdown lint extension, it's quite useful thanks!